games-emulation/melonds/melonds-0.9.5.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="Nintendo DS/DSi emulator"
+DESCRIPTION="MelonDS is an open source and feature-rich Nintendo DS/DSi emulator"
+HOMEPAGE="http://melonds.kuribo64.net/"
+COPYRIGHT="2016-2022 melonDS Team"
+PACKAGER="Luca D'Amico <damico.luca91@live.it>"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/melonDS-emu/melonDS/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="52c6b99340b8bba8c52b11a2242591f05e838c34ddd9ec20dcf1a6039405434a"
+SOURCE_DIR="melonDS-$portVersion"
+PATCHES="enable_wayland_flag.patch
+	melonds-0.9.5.patch"
+ADDITIONAL_FILES="melonds.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	melonDS$secondaryArchSuffix = $portVersion
+	app:melonDS = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libarchive$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libegl$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libslirp$secondaryArchSuffix
+	lib:libX11$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libarchive$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libsdl2_2.0$secondaryArchSuffix
+	devel:libslirp$secondaryArchSuffix
+	libX11${secondaryArchSuffix}_devel
```

Comment:
In this case you might want to use `xlib${secondaryArchSuffix}_devel` instead of `libX11${secondaryArchSuffix}_devel` (not checked).

Reply:
Do you mean xlibe?

Reply:
My bad :) xlibe${seconddaryArchSuffix}_devel indeed.

Reply:
Builds fine with xlibe :)